locales/ku_KU/duckduckgo.po

```diff
@@ -0,0 +1,5940 @@
+
+msgid "Clear your cookies often?"
+msgstr "Kiloran xwe gelek deman pak bike ?"
+
+msgid "Click %s+%s!"
+msgstr "%s+%s! bitikîne"
+
+msgctxt "install-duckduckgo"
+msgid "Click %sAdd Extension%s."
+msgstr "%sPêvekê Tevlî bike%s bitikîne."
+
+msgid "Click %sAdd or change home page...%s"
+msgstr "%sMalpera sereke tevlî bike an jî biguherîne bitikîne...%s"
+
+msgid "Click %sAdd%s"
+msgstr "%sTevlî bike%s bitikîne"
+
+msgctxt "install-duckduckgo"
+msgid "Click %sAllow%s, then %sAdd%s."
+msgstr "%sMafê bide%s bitikîne, paşê %sTevlî bike%s."
+
+msgid "Click %sChange Search Settings%s in the drop down"
+msgstr "%sSazkariyên lêgerînê biguherîne%s bitikîne di menuya daketinê de"
+
+msgid ""
+"Click %sEdit > Preferences%s (on Windows) %sSeaMonkey > Preferences%s (on "
+"Mac)"
+msgstr "%sSerrastkirin > Hilbijartin%s (li ser Windows) %sSeaMonkey > Hilbijartin%s (li ser Mac) "
+
+msgid "Click %sHere%s to add us as a search engine"
+msgstr "%sLi vir%s bitikîne da ku me wekî motora lêgerînê tevlî bikî."
+
+msgid "Click %sHere%s to download the DuckDuckGo extension"
+msgstr "%sLi vir%s bitikîne da ku pêveka DuckDuckGo daxî"
+
+msgid "Click %sOpen%s to download and open the DuckDuckGo Safari extension"
+msgstr "%sVeke%s bitikîne da ku pêvekên DuckDuckGo Safari vekî"
+
+msgid "Click %sPrivacy and Services%s in the left sidebar."
+msgstr "%sTayebetî û Raje%s bitikîne di darika kêlekê de."
+
+msgid ""
+"Click %sSafari%s in the top menu (On Windows, click the %sgears icon%s in "
+"the top right)"
+msgstr "%sSafari%s bitikîne di menuya sereke de (Li ser Windows %sgears icon%s bitikîne di jor rastê de, "
+
+msgid "Click %sSettings%s"
+msgstr "%sSazkariyan%s bitikîne"
+
+msgid "Click %sSettings%s in the dropdown."
+msgstr "%sSazakarî%s bitikîne di menuya daketinê de."
+
+msgid "Click %sUse current pages%s then %sClick OK%s."
+msgstr "%sRûpelên heyê bi kar bîne%s bitikîne paşê  %sBaş e bitikîne%s."
+
+msgid "Click %sYes%s"
+msgstr "%sErê%s bitikîne"
+
+msgid "Click %ssettings/hamburger icon %s on the Chrome toolbar (top right)."
+msgstr "%sSazkariyên/sembola hamburger icon %s bitikîne li ser alavbenda Chrome (jor rastê)."
+
+msgid "Click OK."
+msgstr "Baş e bitikîne."
+
+msgid "Click add."
+msgstr "Tevlî bike bitikîne."
+
+msgid "Click on %sBrowser%s in the sidebar"
+msgstr "Li ser %sGerokê%s bitikîne di darika kêlekê de"
+
+msgid "Click on %sChange Search Settings%s in the dropdown menu"
+msgstr "Li ser %sSazkariyên Lêgerînê Biguherîne%s bitikîne di menuya daketinê de"
+
+msgid "Click on %sSet as Default%s towards the bottom"
+msgstr "%sWekî berdest saz bike%s bitikîne ber bi jêr ve"
+
+msgid "Click on the %sSearch Providers%s under Add-on Types"
+msgstr "%sPeydekarên lêgerînê%s bitikîne di bin cureyên pêvekê de"
+
+msgid "Click on the %sTools icon%s in the top-right of the browser"
+msgstr "%sSembola alavan%s bitikîne di jor rastê ji dîmendera gerokê"
+
+msgid "Click on the %smagnifying glass%s in the search bar"
+msgstr "%smezinker bitikîne di darika lêgerînê de"
```

Comment:
Now this one is missing one placeholder 🙂.

Reply:
@roj1512 will provide missing data soon